pkg/storage/flamegraph.go

```diff
@@ -166,7 +168,7 @@ func GenerateFlamegraph(
 		if it.NextChild() {
 			child := it.At()
 			id := child.LocationID()
-			l, found := locs[id]
+			l, found := locs[string(id[:])]
```

Comment:
It would be nice to have comments for these types of voodoo magics.

Reply:
It's going to be ALL across Parca for the time until we can actually move to [16]byte for the UUIDs across the board, I think.

Reply:
I didn't want to conflate the PR further, I was thinking of subsequently moving the `.LocationID()` functions to be plain `[]byte`. What's happening here is the `uuid.UUID`, which is a `[16]byte`, is being converted to a plain `[]byte`. Since this will be in a quick follow up, would it be ok with you to skip it?

Reply:
Yes, totally ok.